patches/minecraft/net/minecraft/entity/item/EntityItem.java.patch

```diff
@@ -68,23 +68,33 @@
                      else
                      {
                          itemstack1.func_190917_f(itemstack.func_190916_E());
-@@ -282,6 +303,7 @@
+@@ -271,7 +292,8 @@
+         {
+             this.field_70171_ac = false;
+         }
+-
++        
```

Comment:
Please remove this whitespace change.

Reply:
Sorry, I'm having hard time figuring out where exactly these whitespace problems are. I think that field is the inWater field, and I am trying to compare where it is used to the original Forge folder minecraft stuff before I started changing it. Let me keep trying to compare. I can do it but since the Forge minecraft file changes are ignored by git I have to do some sleuthing.

Regarding pinging you in commits, I had no idea that mentioning you in the commit would do that -- kinda cool that it does though. Wanted to make sure the commits were associated with your suggestions, but I'll avoid putting the @ before your name and that should (I think) prevent pings.

Reply:
Yeah, it's weird but for example, in my IDE the original EntityItem code looks like this:
        else
        {
            this.inWater = false;
        }

        return this.inWater;
    }

And my edited version (which should reflect in the patch) looks like this:
        else
        {
            this.inWater = false;
        }
        
        this.inWater = net.minecraftforge.common.ForgeHooks.isInLiquid(this.getEntityBoundingBox(), this);
        return this.inWater;

So it seems to have the same whitespace, except I've inserted the line with the forge hooks. Is it a problem with the "type" of whitespace? Like maybe newline or some extra space or something that isn't easy to see?

Reply:
Okay, yeah it seems that the problem is the type of whitespace. In my new one it has some spaces. I'll fix that. I suppose for patching purposes I should have my IDE mark all the whitespace, so I'll turn that on.